de.metas.acct.base/src/main/java-legacy/org/compiere/acct/Doc_AllocationHdr.java

```diff
@@ -293,6 +317,54 @@ else if (hasPaymentDocument)
 	}   // createFact
 
 	/**
+	 * Create facts for payments in the case when no invoice was involved.
+	 * The pay Amt will go to Credit for outgoing payments and to Debit for Incoming payments
+	 * 
+	 * @param fact
+	 */
+	private void createFacts_PaymentAllocation(final Fact fact)
+	{
+		final MAcctSchema as = fact.getAcctSchema();
+
+		for (int i = 0; i < p_lines.length; i++)
+		{
+			final DocLine_Allocation line = (DocLine_Allocation)p_lines[i];
```

Comment:
i think u shall setC_BPartner_ID(line.getC_BPartner_ID()) .... same as we did for the other case(s)